### DISCUSSION TITLE
Как корректно запустить на Windows 11?

### DISCUSSION THREAD

**Discussion by ThurBiiNah** [hufrea/byedpi#202] [Q&A]
Всем привет! 👋

Скачал архив для Windows x64, распаковал, запустил service_install.bat, в диспетчере задач появился процесс ciapdi.exe, командная строка "ххх\ciadpi.exe" --split 1 --disorder 3+s --mod-http=h,d --auto=torst --tlsrec 1+s"
![image](https://github.com/user-attachments/assets/653c1552-a788-4f88-a381-5983aabf7d10)

В службах также появился ByeDPI с автозапуском
![image](https://github.com/user-attachments/assets/9287152f-eaed-4e4a-a11a-1f87dd2dc81a)

После перезагрузки ПК все также запускается и вроде как работает, че-то куда-то через интернет передает, на скорость спидтеста не влияет, но дискорд не запускается, ютуб тормозит.

Я все правильно сделал? ByeDPI работает как надо, просто вот эти настройки неправильные (--split 1 --disorder 3+s --mod-http=h,d --auto=torst --tlsrec 1+s) ?

Могу менять их на другие, прописывая их в service_install.bat и запуская (в диспетчере и службах настройки меняются), пробовал некоторые из других дискуссий, но ютуб и дискорд также не работают ...

**Comment by 0x131313** [comment]
Чтобы заработало, нужно прописать прокси, который вы подняли - по умолчанию это `socks=127.0.0.1 | port 1080`
Несколько вариантов как это сделать [тут](https://superuser.com/questions/1528185/how-can-i-set-socks-proxy-on-windows/1618473)

Без указания прокси - не заработает.
